addons/account/views/account_view.xml

```diff
@@ -1167,10 +1167,10 @@
                                         <field name="tax_line_id" readonly="1" attrs="{'invisible': [('tax_line_id','=',False)]}"/>
                                         <field name="tax_ids" widget="many2many_tags" readonly="1" attrs="{'invisible': [('tax_ids','=',[])]}"/>
                                     </group>
-                                    <group string="Matching" attrs="{'invisible':[('full_reconcile_id','=',False), ('matched_debit_ids', '=', []),('matched_credit_ids', '=', [])]}">
-                                        <label for="full_reconcile_id"/>
+                                    <group string="Matching" attrs="{'invisible':[('matched_debit_ids', '=', []),('matched_credit_ids', '=', [])]}">
+                                        <label for="full_reconcile_id" attrs="{'invisible':[('full_reconcile_id','=',False)]}"/>
```

Comment:
let's always display the label, if there is partial or full reconcile. That will avoid having a link floating around without a label to attach to. 
So, all in all, the label for full_reconcile_id should have the same visibility as the whole matching group

Reply:
@qdp-odoo ok, attended your comment. Ready!